### PR TITLE
moving secrets off var.tfvars file and to command line args

### PR DESCRIPTION
**Pull request by miyamotoh**
Saving sensitive data in a text file on disk is not a good practice. Moving them off to command line args with `-var` for the `terraform apply` command. Tested fine by building a 4.5 cluster in tor01. Open to different env vars naming and wording for the doc. Just let me know.

Signed-off-by: Hiro Miyamoto <miyamotoh@us.ibm.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/86#" title="Author self-approved">miyamotoh</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **yussufsh**
You can assign the PR to them by writing `/assign @yussufsh` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
Welcome @miyamotoh! It looks like this is your first PR to ocp-power-automation/ocp4-upi-powervs 🎉

**Comment by ltccci** [comment]
Hi @miyamotoh. Thanks for your PR.

I'm waiting for a [ocp-power-automation](https://github.com/orgs/ocp-power-automation/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/ocp-power-automation/ocp4-upi-powervs#contributing) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by yussufsh** [comment]
Good input.

FYI Terraform support passing variables directly from env.
eg: `export TF_VAR_<variable_name>`
https://www.terraform.io/docs/commands/environment-variables.html#tf_var_name

**Comment by miyamotoh** [comment]
Do you want me to change this PR to go with `TF_VAR_xxxxx` (so we can keep the `terraform apply` cmd the same as before)? Let me know.

**Comment by ltccci** [comment]
@miyamotoh: Adding label `do-not-merge/contains-merge-commits` because PR contains merge commits, which are not allowed in this repository.
Use `git rebase` to reapply your commits on top of the target branch. Detailed instructions for doing so can be found [here](https://git.k8s.io/community/contributors/guide/github-workflow.md#4-keep-your-branch-in-sync).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: docs/quickstart.md
Comment (bpradipt) [inline docs/quickstart.md:41]: How about rephrasing it to - 
"You can use environment variables for sensitive variables that should not be saved to disk"

File: docs/quickstart.md
Comment (bpradipt) [inline docs/quickstart.md:54]: Provide both the invocations and you can mention like
"If using environment variables for sensitive data then use the following command"
...

File: docs/var.tfvars-doc.md
Comment (bpradipt) [inline docs/var.tfvars-doc.md:123]: ""This is sensitive data and if you don't want to save it on disk, use environment variables and pass it to `terraform apply` command as shown in .."

File: docs/quickstart.md
Comment (bpradipt) [inline docs/quickstart.md:46]: One problem with this is that it'll be in shell history which is saved in disk :-)
So best solution is to disable history and then export these variables
```
set +o history
export IBMCLOUD_API_KEY=xxxxxxxxxxxxxxx
....
set -o history
```

Comment (miyamotoh) [reply docs/quickstart.md:46]: True! Will be in the next push.

File: docs/quickstart.md
Comment (miyamotoh) [reply docs/quickstart.md:41]: Sure!

File: docs/quickstart.md
Comment (miyamotoh) [reply docs/quickstart.md:54]: Yep, will be in the next push.

File: docs/var.tfvars-doc.md
Comment (miyamotoh) [reply docs/var.tfvars-doc.md:123]: Yep, will be in the next push.

**Review by bpradipt — APPROVED** [review]
/lgtm
Thanks @miyamotoh

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/86#pullrequestreview-494837778" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/86#" title="Author self-approved">miyamotoh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [bpradipt]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
